handler/main.go

```diff
@@ -34,35 +34,53 @@ func eventHandler(ctx context.Context, logsEvent events.CloudwatchLogsEvent) {
 	compressedPayload, err := base64.StdEncoding.DecodeString(cwData)
 	if err != nil {
 		log.Fatalf("error decoding base64 cloudwatch data: %v", err)
+		return
 	}
 
 	r, err := gzip.NewReader(bytes.NewReader(compressedPayload))
 	if err != nil {
 		log.Fatalf("error decompressing cloudwatch data: %v", err)
+		return
 	}
 
 	s, err := ioutil.ReadAll(r)
 	if err != nil {
 		log.Fatalf("error reading decompressed cloudwatch data: %v", err)
+		return
 	}
 
 	payload := make(map[string]interface{})
 
 	err = json.Unmarshal(s, &payload)
 	if err != nil {
 		log.Fatalf("error unmarshalling cloudwatch data to map: %v", err)
+		return
 	}
 
 	log.Printf("***Payload:\n%v", payload)
 
-	logEvents := payload["logEvents"].([]map[string]interface{})
+	logEvents := payload["logEvents"].([]interface{})
```

Comment:
This can still have a runtime panic, it would be better implemented using `val, ok := object.([]interface{})` which is like a try cast, then if `ok` is `false` you can poop out a message.